.pre-commit-config.yaml

```
@@ -0,0 +1,8 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
```

Необходимо добавить локальные хуки (flak8, isort)
А также заполнить `setup.cfg` для flake8
```suggestion
repos:
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v3.2.0
    hooks:
    -   id: trailing-whitespace
    -   id: end-of-file-fixer
    -   id: check-yaml
    -   id: check-added-large-files

- repo: local
  hooks:

    - id: isort
      name: isort
      entry: isort
      files: ^(bot/|core/|src/)
      language: system
      types: [python]
      args: ["--profile", "black", --line-length=79]

    - id: flake8
      name: flake8
      entry: flake8
      files: ^(bot/|core/|src/)
      language: system
      types: [python]
      args: [--config, setup.cfg]

```